chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -6,7 +6,14 @@
 
 mon_init = []
 mon_nodes.each do |monitor|
-    mon_init << monitor.name.split(".")[0]
+  mon_name = monitor.name.split(".")[0]
```

Comment:
Sorry, I had to revert this part, because `get_node_name(monitor)` doesn't work here. You look on gating test here ttps://ci.suse.de/job/openstack-mkcloud/50245/consoleFull. Right now gating tests should pass